docs/build/guides/basics/automate-reset-data.mdx

```diff
@@ -0,0 +1,460 @@
+  }
+}
+
+// Issues an Asset
+async function issueAsset(
+  server,
+  networkPassPhrase,
+  issuerKeys,
+  receivingKeys,
+  customAsset,
+) {
+  try {
+    // First, the receiving account must trust the asset
+    const receiver = await server.getAccount(receivingKeys.publicKey());
+    let transaction = new StellarSdk.TransactionBuilder(receiver, {
+      fee: StellarSdk.BASE_FEE,
+      networkPassphrase: networkPassPhrase,
+    })
+      .addOperation(
+        StellarSdk.Operation.changeTrust({
+          asset: customAsset,
+          limit: "1000",
+        }),
+      )
+      // setTimeout is required for a transaction
+      .setTimeout(100)
+      .build();
+    transaction.sign(receivingKeys);
+    const result = await server.sendTransaction(transaction);
+    await getTxsReports(
+      result,
+      `Receiver Trusting ${customAsset.code} Asset......`,
+      server,
+    );
+
+    // Second, the issuing account actually sends a payment using the asset
+    const issuer = await server.getAccount(issuerKeys.publicKey());
+    transaction = new StellarSdk.TransactionBuilder(issuer, {
+      fee: StellarSdk.BASE_FEE,
+      networkPassphrase: networkPassPhrase,
+    })
+      .addOperation(
+        StellarSdk.Operation.payment({
+          destination: receivingKeys.publicKey(),
+          asset: customAsset,
+          amount: "800", // change to desired amount you want to pay
+        }),
+      )
+      // setTimeout is required for a transaction
+      .setTimeout(100)
+      .build();
+    transaction.sign(issuerKeys);
+    const resultFeeback = await server.sendTransaction(transaction);
+    await getTxsReports(
+      resultFeeback,
+      `Issuer Payment using ${
+        customAsset.code
+      } to  ${receivingKeys.publicKey()}`,
+      server,
+    );
+  } catch (e) {
+    console.error("An error occurred while issuing assets:", e);
+  }
+}
+
+//Create Liquidity Pool
+async function createLiquidityPool(
+  server,
+  accountKeypair,
+  nativeAsset,
+  customAsset,
+  networkPassphrase,
+) {
+  try {
+    const account = await server.getAccount(accountKeypair.publicKey());
+
+    // Create the liquidity pool
+    const poolIdAsset = new StellarSdk.LiquidityPoolAsset(
+      nativeAsset,
+      customAsset,
+      StellarSdk.LiquidityPoolFeeV18,
+    );
+    const poolId = poolIdAsset.toString().split(":")[1]; // To Get the Pool ID
+
+    const transaction = new StellarSdk.TransactionBuilder(account, {
+      fee: StellarSdk.BASE_FEE,
+      networkPassphrase: networkPassphrase,
+    })
+      .addOperation(
+        StellarSdk.Operation.changeTrust({
+          asset: poolIdAsset,
+          limit: "10000", // Set an appropriate limit
+        }),
+      )
+      .addOperation(
+        StellarSdk.Operation.liquidityPoolDeposit({
+          liquidityPoolId: poolId,
+          maxAmountA: "1000", // Amount of asset A to deposit
+          maxAmountB: "1000", // Amount of asset B to deposit
```

Comment:
Since you're going to use `accountTwo` for these LP operations, you will need to modify `maxAmountB`. Your payment operation from earlier only sent 800 to `accountTwo`, so this operation would fail. Either the payment or this deposit op should be changed.